pkg/build/api/validation/validation.go

```diff
@@ -64,14 +64,15 @@ func ValidateBuildConfig(config *buildapi.BuildConfig) field.ErrorList {
 	fromRefs := map[string]struct{}{}
 	specPath := field.NewPath("spec")
 	triggersPath := specPath.Child("triggers")
+	buildFrom := buildutil.GetImageStreamForStrategy(config.Spec.Strategy)
```

Comment:
I don't really want to do a migration yet.

On Thu, Feb 4, 2016 at 6:27 PM, Ben Parees <notifications@github.com> wrote:

> In pkg/build/api/validation/validation.go
> <https://github.com/openshift/origin/pull/6952#discussion_r51955363>:
>
> > @@ -64,14 +64,15 @@ func ValidateBuildConfig(config *buildapi.BuildConfig) field.ErrorList {
> >  	fromRefs := map[string]struct{}{}
> >  	specPath := field.NewPath("spec")
> >  	triggersPath := specPath.Child("triggers")
> > +	buildFrom := buildutil.GetImageStreamForStrategy(config.Spec.Strategy)
>
> @smarterclayton <https://github.com/smarterclayton> would that (adding a
> conversion) be the appropriate way to tackle this, or should it be part of
> an etcd migration tool?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/6952/files#r51955363>.
>